osquery/tables/system/darwin/signature.mm

```diff
@@ -23,6 +22,34 @@
 namespace osquery {
 namespace tables {
 
+// Get the flags to pass to SecStaticCodeCheckValidityWithErrors, depending on
+// the OS version.
+Status getVerifyFlags(SecCSFlags& flags) {
```

Comment:
What's the threading story in `osquery`?  That would be much nicer, if it won't cause concurrency issues!